lib/api/build.rs

```diff
@@ -274,7 +274,13 @@ fn configure_validation(builder: Builder) -> Builder {
             ("QueryBatchPoints.timeout", "custom(function = \"crate::grpc::validate::validate_u64_range_min_1\")"),
             ("FacetCounts.collection_name", "length(min = 1, max = 255)"),
             ("FacetCounts.key", "length(min = 1)"),
-            ("FacetCounts.timeout", "custom(function = \"crate::grpc::validate::validate_u64_range_min_1\")")
+            ("FacetCounts.filter", ""),
```

Comment:
added missing validation on `FacetCounts.filter`

Reply:
Thank you!